Pear/Settings/EditingViewController.swift

```diff
@@ -180,24 +180,14 @@ class EditingViewController: UIViewController {
 
     private func setupSections() {
         if isShowingGroups {
-//            setupSectionsFomGroups()
-            let organizationStrings = Constants.Options.organizations.map(\.name)
-            let yoursAndMore = removeDuplicates(yourStrings: user.groups, moreStrings: organizationStrings)
-            let stringToGroup = { ItemType.group(Constants.Options.organizationsMap[$0]!) }
-            sections = [
-               Section(type: .yours, items: yoursAndMore.your.compactMap(stringToGroup)),
-               Section(type: .more, items: yoursAndMore.more.compactMap(stringToGroup))
-            ]
+            setupSectionsFomGroups()
         } else {
-//            setupSectionsFromInterests()
-            let interestsStrings = Constants.Options.interests.map(\.name)
-            let yoursAndMore = removeDuplicates(yourStrings: user.interests, moreStrings: interestsStrings)
-            let stringToInterest = { ItemType.interest(Constants.Options.interestsMap[$0]!) }
-            sections = [
-                Section(type: .yours, items: yoursAndMore.your.compactMap(stringToInterest)),
-                Section(type: .more, items: yoursAndMore.more.compactMap(stringToInterest))
-            ]
+            setupSectionsFromInterests()
         }
+        sections = [
+            Section(type: .yours, items: []),
+            Section(type: .more, items: [])
+        ]
```

Comment:
It seems to me that this can only ever be either (a) an empty array, or (b) an array of two elements, `.yours` and `.more`. Is this correct? If it is this structured, then I would definitely use a different data structure, like an enum or an optional tuple.